category.php

```diff
@@ -0,0 +1,56 @@
+<?php
+get_header(); ?>
+
+<section class="quote">
+  <div class="quote__inner">
+    <h2 class="quote__title">Quotey mCQuoterson</h2>
+    <h2 class="quote__author">Quote Author</h2>
+  </div>
+</section>
+
+<div class="container">
+	<h2 class="heading--one heading--bold heading--primary u-align-center"><?php single_cat_title(); ?></h2>
+
+	<section class="blog">
+		<ul class="grid grid--space-bottom">
+
+		    <?php
+	        // the query
+	        // TODO: Ideally category name would be a dynamic value.
+	        $the_query = new WP_Query(array('category_name' => 'Jummah', 'post_status' => 'publish', 'posts_per_page' => 5,
```

Comment:
so I believe all you should need to do here is replace this custom query with the default WordPress one as it should already know which category this page is (hence why `single_cat_title` outputs correctly (or at least it should!))

i.e.

```
if (have_posts()):
    while (have_posts()):
        the_post();
    endwhile;
endif;
```

should also be able to remove the `wp_reset_postdata()`'s too as will just be a normal query